requirements.txt

```diff
@@ -15,6 +14,16 @@ Flask-Login==0.6.3
 future==1.0.0
 marshmallow==3.21.1
 numpy==1.26.4
+# Need to run 'opentelemetry-bootstrap -a install' command and it is a CLI tool and should be run seperately
```

Comment:
Can this make sure to clarify that this command gets run AFTER the pip install is done? Actually, not even sure it should be here, it should be in some sort of setup documentation probably, or at least in both places.

Reply:
Adding clarifying language to this and `CONTRIBUTING.md` 👍